tests/bridge/test_bridge.py

```diff
@@ -65,6 +66,105 @@
 ]
 
 
+#there's probably a better way if importing these addresses - export to a config file or something?
+coreContract = "0x2A8facc9D49fBc3ecFf569847833C380A13418a8"
```

Comment:
These should be added to the eth registry in `helpers/registry/eth_registry.py`. Some of these addresses already exist there.

Good practice to grep the repo for addresses/constants to see if they already exist.